tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/date_and_time.py

```diff
@@ -21,3 +30,45 @@ def __init__(self, marionette):
     def select_time_format(self, time_format):
         self.marionette.find_element(*self._24h_selector_locator).tap()
         self.select(time_format)
+
+    def change_automatic_time_update(self, state):
+
+        Wait(self.marionette).until(expected.element_displayed(
+            Wait(self.marionette).until(
+                expected.element_present(*self._autotime_enabled_locator))))
+
+        self.marionette.find_element(*self._autotime_enabled_switch_locator).tap()
+
+        if not state:
+            self.wait_for_condition(lambda m: self.is_autotime_enabled is False)
+        else:
+            self.wait_for_condition(lambda m: self.is_autotime_enabled is True)
+
+    def set_region(self, region):
+
+        self.marionette.find_element(*self._timezone_region_locator).tap()
+        self.marionette.switch_to_frame()
+        self.wait_for_condition(
+            lambda m: len(self.marionette.find_elements(*self._timezone_selection_locator)) > 0)
+
+        options = self.marionette.find_elements(*self._timezone_selection_locator)
+        close_button = self.marionette.find_element(*self._timezone_confirm_button_locator)
+
+        # loop options until we find the match
+        for li in options:
+            if region in li.text:
+                li.tap()
+                break
+        else:
+            raise Exception("Element '%s' could not be found in select wrapper" % region)
+        close_button.tap()
```

Comment:
And perhaps move the 
+ close_button = self.marionette.find_element(*self._timezone_confirm_button_locator)
call just before it?